x-pack/plugins/observability_solution/entity_manager/common/constants_entities.ts

```diff
@@ -5,17 +5,20 @@
  * 2.0.
  */
 
-export const ENTITY_VERSION = 'v1';
-export const ENTITY_BASE_PREFIX = '.entities-observability';
-export const ENTITY_HISTORY_BASE_PREFIX = `${ENTITY_BASE_PREFIX}.history-${ENTITY_VERSION}`;
-export const ENTITY_LATEST_BASE_PREFIX = `${ENTITY_BASE_PREFIX}.latest-${ENTITY_VERSION}`;
-export const ENTITY_HISTORY_TRANSFORM_PREFIX = `entity-history-${ENTITY_VERSION}`;
-export const ENTITY_LATEST_TRANSFORM_PREFIX = `entity-latest-${ENTITY_VERSION}`;
+export const ENTITY_BASE_PREFIX = 'entities';
+export const ENTITY_SCHEMA_VERSION_V1 = 'v1';
+export const ENTITY_HISTORY = 'history';
+export const ENTITY_LATEST = 'latest';
+export const ENTITY_INDEX_PREFIX = `.${ENTITY_BASE_PREFIX}`;
```

Comment:
Based on the announced changes by the ES team, we should likely remove the dot from here.
The question is if we should pair that change with making the indices hidden?

Reply:
👍 

we can review this as-is and continue the discussion on dot-prefix in isolation